### DISCUSSION TITLE
When viewing a patch file, browser search highlight is missing on changed lines

### DISCUSSION THREAD

**Discussion by dagood** [community/community#76521] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When I view a patch file, the green and red highlighting prevents the browser's search highlight from showing up. This can make it very hard to quickly find a specific change.

Example for the screenshots below: https://github.com/microsoft/go/blob/2600c2eb59104786780bf105cd086417568b3df7/patches/0005-Add-CNG-crypto-backend.patch#L167-L171

Microsoft Edge for Business
Version 119.0.2151.72 (Official build) (64-bit)
Windows 11

![image](https://github.com/community/community/assets/12819531/22bb5daa-5c66-4a4e-9842-d8286b9d5cb7)

![image](https://github.com/community/community/assets/12819531/4225892f-4efb-4999-86b1-c68beb69bac0)

Firefox 120.0 (64-bit)
Windows 11

![image](https://github.com/community/community/assets/12819531/95bcd728-88fc-4ca4-9a7e-062cd5a0982d)

**Comment by Mohammad222PR** [comment]
It seems like you're experiencing an issue with viewing patch files on GitHub using Microsoft Edge for Business. While I can't directly solve browser-related issues, I can suggest a couple of things you might try:

Use a Different Browser: Try opening the same patch file in a different browser, such as Google Chrome or Mozilla Firefox, to see if the issue persists. This can help determine if the problem is specific to Microsoft Edge.

Browser Extensions: Disable browser extensions or add-ons one by one to see if any of them are causing the conflict with the highlighting. Sometimes, certain extensions can interfere with the default behavior of web pages.

Update Browser: Ensure that your Microsoft Edge browser is up-to-date. Check for any available updates and install them.

GitHub Dark Mode: If you're using GitHub Dark Mode, try switching back to the light mode temporarily to see if the issue persists.

If the problem continues, you may want to consider reporting this issue to GitHub or Microsoft Edge support, providing the details you've mentioned here. They may be able to provide more specific assistance or address the bug in future updates
